logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -13,6 +13,7 @@ function Get-DefaultDevelopmentSettingsByProject {
             Urls              = "http://localhost:54746"
             ApiSettings       = @{
                 Engine = ""
+                DecryptSecrets = $true
```

Comment:
I would call it PlainTextSecret - This feature either stores hash or plaintext secrets, there is no encryption going on here.

Reply:
done